networking/network_observability/network-observability-network-policy.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="network-observability-network-policy"]
+= Network Policy
+include::_attributes/common-attributes.adoc[]
+:context: network_observability
+
+toc::[]
+
+As a user with the admin role, you can create a network policy for a namespace.
```

Comment:
@memodi, can't users with project admin also create network policies in that namespace?

Reply:
@jab-rh workloads under netobserv (created via by flowcollector CR) can only be created by cluster admin roles, so in this case a cluster admin would also be project admin.